index.src.html

```diff
@@ -1497,10 +1537,11 @@ <h4 algorithm id="collectfromcredentialstore-federatedcredential">
   </ol>
 
   <h4 algorithm id="create-federatedcredential">
-    `FederatedCredential`'s `[[Create]](options)`
+    `FederatedCredential`'s `[[Create]](origin, options)`
   </h4>
 
-  <dfn for="FederatedCredential" method>\[[Create]](options)</dfn> is called with a
+  <dfn for="FederatedCredential" method>\[[Create]](origin, options)</dfn> is called with 
+  an [=environment settings object/origin=] (<var ignore>origin</var>) and a
```

Comment:
Likewise, this needs to make its way through to https://w3c.github.io/webappsec-credential-management/#abstract-opdef-create-a-federatedcredential-from-federatedcredentialinit.

Reply:
ok, will look into it.

Reply:
Ditto.